inc/main.php

```diff
@@ -144,6 +145,26 @@ public static function change_lazyload_default( $defaults ) {
 		return $defaults;
 	}
 
+	/**
+	 * Change limits default for new users.
+	 *
+	 * @param array $defaults Old defaults.
+	 *
+	 * @return array New defaults.
+	 */
+	public static function change_limits_default( $defaults ) {
+		$install_time = get_option( 'optimole_wp_install', 0 );
+
+		// Todo: change this before release to release day.
+		if ( $install_time < 1683870956 ) {
```

Comment:
@abaicus I would use the timestamp here since not all the people update in the same time and is not that reliable. We can use if the user is connected or not, if is conected we consider old, if he is not connected we consider new. @GrigoreMihai you used a similar approach on noscript approach

Reply:
@selul Thanks for letting me know. I changed the way we treat the setting based on the connection status.